nni/compression/pytorch/speedup/compressor.py

```diff
@@ -42,18 +42,23 @@ class ModelSpeedup:
         the index of batch dimension in the dummy_input
     confidence: the confidence coefficient of the sparsity inference. This value is
         actually used as the batchsize of the dummy_input.
+    customized_replace_func: None/Dict
+        If the parameter is not None, then we will use the given function to replace the
+        corresponding modules. The `key` of the dict is the opertor types and the `value`
+        is the replace function of corresponding opertor.
```

Comment:
maybe users don't know how to use it if they didn't see the source code, do we need a simple example to show how to write `customized_replace_func`?